cachecontrol/adapter.py

```diff
@@ -117,8 +117,8 @@ def _update_chunk_length(self):
 
         # See if we should invalidate the cache.
         if request.method in self.invalidating_methods and resp.ok:
-            cache_url = self.controller.cache_url(request.url)
-            self.cache.delete(cache_url)
+            cache_key = self.controller.cache_key(request)
```

Comment:
`self.invalidating_methods` shouldn't be extended with `PATH` ? We're using this method for updating user profile.